google-cloud-bigquery/test/google/cloud/bigquery/model/columns_test.rb

```diff
@@ -25,7 +25,20 @@
   let(:model) { Google::Cloud::Bigquery::Model.from_gapi_json model_full_hash, bigquery.service }
 
   it "maps columns to primitive ruby values" do
-    model.feature_columns.must_equal [{ name: "f1", type: { typeKind: "STRING" } }]
-    model.label_columns.must_equal [{ name: "predicted_label", type: { typeKind: "FLOAT64" } }]
+    model.feature_columns.count.must_equal 1
```

Comment:
@shollyman Can you verify this is the type of structure you were looking for regarding the Standard SQL fields and types?